examples/miscellaneous/miscellaneous_ex2/run.sh

```diff
@@ -6,4 +6,4 @@ source $LIBMESH_DIR/examples/run_common.sh
 
 example_name=miscellaneous_ex2
 
-run_example "$example_name" -f .5
+run_example "$example_name" -f .5 0.1 i
```

Comment:
I guess the "i" is supposed to be sort of like a comment here?  It was more like a red herring to me; I went back to hunt for what exactly the code was doing with argv[4]...

Reply:
oh yes, it is rather an artefact; initially the i was read as well. But lets remove it.